vibi-dpu/src/core/approval.rs

```diff
@@ -24,7 +25,21 @@ pub async fn process_approval(deserialised_msg_data: &Value) {
         return;
     }
     let relevance_vec = relevance_vec_opt.to_owned().expect("Empty coverage_opt");
-    let coverage_map_obj = CoverageMap::calculate_coverage_map(repo_provider, reviewer_handles, relevance_vec);
+    let mut coverage_map_obj = CoverageMap::new(repo_provider);
+    coverage_map_obj.calculate_coverage_map(relevance_vec, reviewer_handles);
     // add up contribution of aliases
     // add comment
+    let comment_text = approval_comment_text(&coverage_map_obj);
+    // get access token and call add_comment in gh/bb
+}
+
+fn approval_comment_text(coverage_map: &CoverageMap) -> String {
+    let mut comment = "Relevant users for this PR:\n\n".to_string();  // Added two newlines
```

Comment:
We should ideally start by telling the right relevant reviewer approval percentage of the PR.
Let me know if that value is not available to us here.

Reply:
Sounds correct. Can do it